File: CodeSense.Domain/Entities/Address.cs
using CodeSense.Domain.Abstractions;

namespace CodeSense.Domain.Entities
{
    public class Address : EntityBase
    {
        public int ClientCompanyId { get; set; }
        public string Street { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string ZipCode { get; set; }
        public string Country { get; set; }
        public bool IsPrimary { get; set; }

        // navigational properties
        public Company ClientCompany { get; set; }
    }
}

File: CodeSense.Host/CodeSense-Project-Management/src/app/interfaces/aggregates/company/address.ts
import { EUCountries } from "src/app/enums/eu-countries";

export interface Address {
    street: string;
    houseNumber: string;
    box?: string;
    city: string;
    state: string;
    zipCode: string;
    country: EUCountries ;
    isPrimary?: boolean;

}
